### PR TITLE
Task/13145 node@16 embercli@3.28

### PR DESCRIPTION
**Pull request by TangoYankee**
# Description
Upgrade to ember-cli 3.28 and node@16
**Depends on** update to ose-ember-addons from [PR 9](https://github.com/NYCPlanning/ose-ember-addons/pull/9)

## Tickets
Closes [AB#13145](https://dev.azure.com/NYCPlanning/cc280b0d-40a0-4689-b852-2e6247f1af50/_workitems/edit/13145)

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 [build]
 command = "yarn build"
-environment = { NODE_VERSION = "12.18.4" }
+environment = {}
```
Comment (TylerMatteo) [inline netlify.toml:3]: Can you just delete these environment object entirely?

File: app/controllers/application.js [hunk 2]
```diff
@@ -91,7 +93,7 @@ export default class ApplicationController extends ParachuteController {
 
   @computed('lat', 'lng')
   get center() {
-    return [this.get('lat'), this.get('lng')];
+    return [this.get('lng'), this.get('lat')];
```
Comment (TylerMatteo) [inline app/controllers/application.js:96]: Is this fixing a bug you noticed along the way? We've had issues keeping these straight in the app in the past.

**Review by TylerMatteo — CHANGES_REQUESTED** [review]

File: app/controllers/application.js (hunk 2 above)
Comment (TangoYankee) [reply app/controllers/application.js:96]: Yes. The behavior was noticeable when the application first loaded. The viewport would be somewhere in the Atlantic and then jump to lower Manhattan. I believe it was trying to start in Africa but was clamped to the local water by the map's bounds.

File: netlify.toml (hunk 1 above)
Comment (TangoYankee) [reply netlify.toml:3]: Ugh, yeah this is literally the same mistake I made in waterfront access and then copied over

Comment (TangoYankee) [reply netlify.toml:3]: As I'll be in here anyway, what would you think of refactoring it in the style of [this Zola POC for netlify](https://github.com/NYCPlanning/labs-zola/pull/1132/files)

Comment (TylerMatteo) [reply netlify.toml]: I'd rather we didn't. I need to think about how we want that to look a little more so for now let's keep the scope of the PR as small as possible.

**Review by TylerMatteo — APPROVED** [review]
Approved pending those tweaks to `netlify.toml`

**Pull request merged**